artview/components/correlation.py

```diff
@@ -642,12 +641,25 @@ def _update_plot(self):
                 self.Vradar.value, self.VfieldHorizontal.value,
                 self.VfieldVertical.value, sweeps, gatefilter, self.ax,
                 vmin + 0.05 * (vmax-vmin), vmax - 0.05 * (vmax-vmin),
-                str(self.horizontal_scale_menu_group.checkedAction().text()),
-                str(self.vertical_scale_menu_group.checkedAction().text()),
+                xscale,
+                yscale,
                 color=self.parameters["color"])
 
         self.canvas.draw()
 
+    def get_xyscale(self):
+        scale = [None, None]
```

Comment:
Why bother with creating this list and then filling? Could just create the list.